contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -435,14 +505,87 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     function setIncentiveSettings(IncentiveSettings memory _newIncentiveSettings) external onlyOperator noRebalanceInProgress {
         incentive = _newIncentiveSettings;
 
-        _validateSettings(methodology, execution, incentive);
+        _validateNonExchangeSettings(methodology, execution, incentive);
 
         emit IncentiveSettingsUpdated(
             incentive.etherReward,
             incentive.incentivizedLeverageRatio,
             incentive.incentivizedSlippageTolerance,
-            incentive.incentivizedTwapCooldownPeriod,
-            incentive.incentivizedTwapMaxTradeSize
+            incentive.incentivizedTwapCooldownPeriod
+        );
+    }
+
+    /**
+     * OPERATOR ONLY: Add a new enabled exchange for trading during rebalances. Note: must not be in a rebalance.
+     *
+     * @param _exchangeName         Name of the exchange
+     * @param _exchangeSettings     Struct containing exchange parameters
+     */
+    function addEnabledExchange(
+        string memory _exchangeName,
+        ExchangeSettings memory _exchangeSettings
+    )
+        external 
+        onlyOperator 
+        noRebalanceInProgress
+    {
+        require(exchangeSettings[_exchangeName].twapMaxTradeSize == 0, "Exchange already enabled");
+        _validateExchangeSettings(_exchangeSettings);
+        
+        exchangeSettings[_exchangeName] = _exchangeSettings;
```

Comment:
It won't cause any weird state, but think we should do this to make the code 100% clear

Reply:
Should we explicitly set the timestamp to 0 when adding a new exchange? I think it would make the most sense since we might want to use an exchange immediately after adding it

Reply:
yeah that makes sense for a new exchange. For changing existing exchanges, imagine it would just use the existing timestamp.

Reply:
I'm fine with a new exchange starting at 0. Where it gets iffy is when we are updating an exchange do we reset the timestamp? If we do then I don't think we want to allow exchange updates to happen during a rebalance in case we accidentally trade during a cool down period.